### PR TITLE
Improve weekly report generated

### LINKED ISSUE #71: Display associate name instead of github acronym
Display the associate name instead of showing the github acroynm.

### PR DESCRIPTION
**Pull request by jacobdotcosta**
## TODO

Improve weekly report generated to 
- [x] Display the full user name and not our github acronym: cmoulliard -> Charles Moulliard within the report - https://github.com/snowdrop-bot/snowdrop-bot/issues/71
- [x] Rename the title to "Weekly Report - 22"
- [x] Sort the issues / project and/or module and status
- [x] Display the status using colors and not using black text - `[closed]`

### CODE REVIEW

File: src/main/java/io/snowdrop/reporting/weekreport/WeekReportEndpoint.java [hunk 1]
```diff
@@ -81,50 +84,63 @@ private void populate(Date startTime, Date endTime) {
...
-      weekNumber = WEEK_YEAR_FORMAT.format(endTime);
+      final Date startTime = parseDate(startTimeString);
+      final Date endTime = parseDate(endTimeString);
+      final String weekNumber = WEEK_YEAR_FORMAT.format(endTime);
       LOGGER.debug("week number: {}", weekNumber);
-      populate(startTime, endTime);
-      mdText = WeeklyDevelopmentReportImpl.build(startTime, endTime, users).buildWeeklyReport();
-      LOGGER.info("weekly_development: {}", mdText);
-      return mdText;
+      final String reportName = String.format(reportNameTemplate, weekNumber);
```
Comment (iocanel) [inline src/main/java/io/snowdrop/reporting/weekreport/WeekReportEndpoint.java:120]: Let's remove commented out code.

File: src/main/java/io/snowdrop/reporting/weekreport/WeeklyDevelopmentReportImpl.java [hunk 2]
```diff
@@ -57,37 +60,37 @@ public static WeeklyDevelopmentReportImpl build(final Date pStartDate, final Dat
...
+      eachAssignee.getValue().stream().collect(Collectors.groupingBy(Issue::getLabel, TreeMap::new, Collectors.toSet())).entrySet().forEach(eachLabel -> {
         String label = eachLabel.getKey();
         labelUnorderedList.getItems().add(label);
         UnorderedList issueUnorderedList = new UnorderedList();
         eachLabel.getValue().stream().forEach(eachIssue -> {
           TextBuilder issueTextB = new TextBuilder();
           Date dateCreatedAt = eachIssue.getCreatedAt();
-          String strMdColor = "gray";
-          if(eachIssue.isOpen()) {
+          String strMdColor = " ![#a9a9a9](https://via.placeholder.com/15/a9a9a9/000000?text=+) ";
```
Comment (iocanel) [inline src/main/java/io/snowdrop/reporting/weekreport/WeeklyDevelopmentReportImpl.java]: Can we please externalize those in application.properties?

**Review by iocanel — CHANGES_REQUESTED** [review]
Looks great!
Added some minor comments.

File: src/main/java/io/snowdrop/reporting/weekreport/WeekReportEndpoint.java (hunk 1 above)
Comment (jacobdotcosta) [reply src/main/java/io/snowdrop/reporting/weekreport/WeekReportEndpoint.java:120]: ...again... :-(

Thanks!

File: src/main/java/io/snowdrop/reporting/weekreport/WeeklyDevelopmentReportImpl.java (hunk 2 above)
Comment (jacobdotcosta) [reply src/main/java/io/snowdrop/reporting/weekreport/WeeklyDevelopmentReportImpl.java]: Done.

File: src/main/java/io/snowdrop/reporting/weekreport/WeekReportEndpoint.java (hunk 1 above)
Comment (jacobdotcosta) [reply src/main/java/io/snowdrop/reporting/weekreport/WeekReportEndpoint.java:120]: Done.

**Review by iocanel — APPROVED** [review]
LGTM

**Pull request merged**